pages/index.vue

```diff
@@ -1,3 +1,35 @@
 <template>
-    <h1>Hello world!</h1>
-</template>
\ No newline at end of file
+  <div class="my-team">
+    <header class="my-team__main-title">
+      <h1>The Creative crew</h1>
+    </header>
+    <div class="my-team__description">
+      <h2>Who we are</h2>
+      <p>
+        We are a team of creatively diverse. driven. innovative individuals
+        working in various locations from the world
+      </p>
+    </div>
+    <MyTeamMembers />
```

Comment:
Suggestion: Add props to `<MyTeamMembers />`  and pass `members` data from fetch. So you don't need to do another fetch in your child component

Reply:
I added props and remove the re-fetch issue